js_modules/dagster-ui/packages/ui-components/src/components/ToggleButton.tsx

```diff
@@ -0,0 +1,28 @@
+import {Icon} from './Icon';
+import {UnstyledButton} from './UnstyledButton';
+
+type Props = {
+  isOpen: boolean;
+  onToggle: (e: React.MouseEvent<HTMLButtonElement>) => void;
+};
+export const ToggleButton = ({isOpen, onToggle}: Props) => {
+  return (
+    <UnstyledButton
+      onClick={(e) => {
+        onToggle(e);
+      }}
+      onKeyDown={(e) => {
+        if (e.code === 'Space') {
+          // Prevent the default scrolling behavior
+          e.preventDefault();
+        }
```

Comment:
Should this be something we do on all buttons actually?

Maybe we want space to toggle the toggle button?